### ISSUE TITLE
A string containing "/*" causes indentation to be messed up

### ISSUE THREAD

**Issue by paholg** [tminor/jsonnet-mode#9]
`/*` inside a string is treated as though it begins a multiline comment for indentation purposes.

Example:

```
{
  foo = "/dir/*",
              bar = "should not be indented",
              baz = "*/ <--- this will end the extra indentation",
  quz = "back to where it should be",
}
```

**Comment by anguslees** [comment]
After a few minutes looking over the code, I'm guessing this is because `jsonnet--find-current-block-comment` doesn't check to see if it is in a string (or other non-trivial context).  I suggest `jsonnet--find-current-block-comment` should be rewritten to use syntax properties - it will be faster and more robust.

**Comment by anguslees** [comment]
Yep, this fixes it, when applied on top of #8 (which is why I haven't just put up a new PR here):

```emacs
(defun jsonnet--find-current-multiline-string ()
  "Return the position of the beginning of the current multiline string.

If not inside of a multiline string, return nil."
  (let* ((ppss (syntax-ppss))
	 (in-string (nth 3 ppss))
	 (start (nth 8 ppss)))
    (when in-string
      start)))
```

**Comment by tminor** [comment]
@anguslees @paholg This issue is very old but I'd be happy to merge any PRs that address it but it's worth pointing out that #17 fixes this problem (and many more!). I plan to merge it soon!
